docs/src/hal/tutorial.adoc

```diff
@@ -422,7 +438,54 @@ You can build very complex HAL systems without ever using a graphical
 interface. However there is something satisfying about seeing the
 result of your work. The first and simplest GUI tool for the HAL is
 halmeter. It is a very simple program that is the HAL equivalent of the
-handy Fluke multimeter (or Simpson analog meter for the old timers).
+handy multimeter (or analog meter for the old timers).
+
+It allows to observe the pins, signals or parameters by displaying the
+current value of these entities. It is very easy to use application for graphical environments. In a console type:
+
+----
+halmeter
+----
```

Comment:
This section is kind of a duplicate with http://linuxcnc.org/docs/devel/html/hal/tools.html#sec:halmeter. We should merge this (manually) at some point.

Also the Halscope section should be moved there.